packages/phrases-ui/src/locales/ru.ts

```diff
@@ -45,8 +45,8 @@ const translation = {
     reminder: 'Напоминание',
     not_found: '404 Не найдено',
     agree_with_terms: 'Я прочитал и согласен с ',
-    agree_with_terms_modal: 'Чтобы продолжить, пожалуйста, согласитесь с <link></link>.',
-    terms_of_use: 'Условия использования',
```

Comment:
I think this line needs to be in sentence case, at least that's what it is in English scripts. So why do you change the first capital letter?